libs/login/feature/src/lib/login.component.ts

```diff
@@ -63,19 +63,17 @@ export class LoginComponent implements OnInit {
     });
 
     PushNotifications.addListener('registration', (token: Token) => {
-      console.log(token.value);
-      alert('Push registration success, token: ' + token.value);
       this.fcmToken = token.value;
     });
 
     PushNotifications.addListener('registrationError', (error: JSON) => {
-      alert('Error on registration: ' + JSON.stringify(error));
+      console.log('Error on registration: ' + JSON.stringify(error));
     });
 
     PushNotifications.addListener(
       'pushNotificationReceived',
       (notification: PushNotificationSchema) => {
-        alert('Push received: ' + JSON.stringify(notification));
+        console.log('Push received: ' + JSON.stringify(notification));
```

Comment:
Console Log

Reply:
This is like the api calls where you log the data/res, otherwise, we get those warnings.